pcp_def_metrics.c

```diff
@@ -247,7 +255,7 @@ void pcp_def_cpu_metrics(struct activity *a)
 
 				else if (a->id == A_IRQ) {
 					/* Create per-CPU interrupts metrics */
-					pcp_def_percpu_int_metrics(a, indom);
+					pcp_def_percpu_int_metrics(a, i - 1);
```

Comment:
Since "first" variable will be set to FALSE, instances will never be created for other CPU that CPU0. Call to pcp_def_percpu_int_metrics() should be moved above before the test on "first" variable.